test/e2e/virtual/syncer/virtualworkspace_test.go

```diff
@@ -1619,12 +1619,19 @@ func TestUpsyncerVirtualWorkspace(t *testing.T) {
 				}
 
 				logWithTimestamp(t, "Creating PV test-pv through upsyncer virtual workspace...")
-				_, err = kubelikeSyncerVWClient.CoreV1().PersistentVolumes().Cluster(workspaceName).Create(ctx, pv, metav1.CreateOptions{})
+				pv, err = kubelikeSyncerVWClient.CoreV1().PersistentVolumes().Cluster(workspaceName).Create(ctx, pv, metav1.CreateOptions{})
+				require.NoError(t, err)
+
+				pv.Status.Phase = corev1.VolumeAvailable
```

Comment:
Can you check here that the status is empty ?
And move this line after the next log message (updating status ...).

Reply:
done!